eslint.config.mjs

```diff
@@ -0,0 +1,46 @@
+import { FlatCompat } from '@eslint/eslintrc'
+import js from '@eslint/js'
+import typescriptEslint from '@typescript-eslint/eslint-plugin'
+import tsParser from '@typescript-eslint/parser'
+import globals from 'globals'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+// This is in place until https://github.com/typescript-eslint/typescript-eslint
```

Comment:
DO we have a ticket for this so we can reference this as a TODO?

Reply:
good idea, I'll add one.